frappe/push_notification.py

```diff
@@ -0,0 +1,285 @@
+		"""
+		Set the project name.
+
+		:param project_name:  (str) The name of the project.
+		:return:
+		"""
+		PushNotification.project_name = project_name
+
+	def add_token(self, user_id: str, token: str) -> tuple[bool, str]:
+		"""
+		Add a token for a user.
+
+		:param user_id: (str) The ID of the user. This should be user's unique identifier.
+		:param token: (str) The token to be added.
+		:return: tuple[bool, str] First element is the success status, second element is the message.
+		"""
+		data = self._send_post_request(
+			"notification_relay.api.token.add", {"user_id": user_id, "fcm_token": token}
+		)
+		return data["success"], data["message"]
+
+	def remove_token(self, user_id: str, token: str) -> tuple[bool, str]:
+		"""
+		Remove a token for a user.
+
+		:param user_id: (str) The ID of the user. This should be user's unique identifier.
+		:param token: (str) The token to be removed.
+		:return: tuple[bool, str] First element is the success status, second element is the message.
+		"""
+		data = self._send_post_request(
+			"notification_relay.api.token.remove", {"user_id": user_id, "fcm_token": token}
+		)
+		return data["success"], data["message"]
+
+	def add_topic(self, topic_name: str) -> bool:
+		"""
+		Add a notification topic.
+
+		:param topic_name: (str) The name of the topic.
+		:return: bool True if successful, False otherwise.
+		"""
+		data = self._send_post_request("notification_relay.api.topic.add", {"topic_name": topic_name})
+		return data["success"]
+
+	# Remove Topic
+	def remove_topic(self, topic_name: str) -> bool:
+		"""
+		Remove a notification topic.
+
+		:param topic_name: (str) The name of the topic.
+		:return: bool True if successful, False otherwise.
+		"""
+		data = self._send_post_request("notification_relay.api.topic.remove", {"topic_name": topic_name})
+		return data["success"]
+
+	def subscribe_topic(self, user_id: str, topic_name: str) -> bool:
+		"""
+		Subscribe a user to a topic.
+
+		:param user_id: (str) The ID of the user. This should be user's unique identifier.
+		:param topic_name: (str) The name of the topic. This topic should be already created.
+		:return:
+		"""
+		data = self._send_post_request(
+			"notification_relay.api.topic.subscribe", {"user_id": user_id, "topic_name": topic_name}
+		)
+		return data["success"]
+
+	# Unsubscribe Topic (User)
+	def unsubscribe_topic(self, user_id: str, topic_name: str) -> bool:
+		"""
+		Unsubscribe a user from a topic.
+
+		:param user_id: (str) The ID of the user. This should be user's unique identifier.
+		:param topic_name: (str) The name of the topic. This topic should be already created.
+		:return: bool True if successful, False otherwise.
+		"""
+		data = self._send_post_request(
+			"notification_relay.api.topic.unsubscribe", {"user_id": user_id, "topic_name": topic_name}
+		)
+		return data["success"]
+
+	def send_notification_to_user(
+		self,
+		user_id: str,
+		title: str,
+		content: str,
```

Comment:
```suggestion
		body: str,
```
Should we call this `body`? I think FCM uses body param so we can keep it consistent.
`content` and `data` are confusing for me